docs/specs/avm-transaction-serialization.md

```diff
@@ -246,6 +246,88 @@ Let’s make a transferable operation:
 
 Outputs have four possible types: [`SECP256K1TransferOutput`](avm-transaction-serialization.md#secp256k1-transfer-output), [`SECP256K1MintOutput`](avm-transaction-serialization.md#secp256k1-mint-output), [`NFTTransferOutput`](avm-transaction-serialization.md#nft-transfer-output) and [`NFTMintOutput`](avm-transaction-serialization.md#nft-mint-output).
 
+## SECP256K1 Mint Output
+
+A [secp256k1](cryptographic-primitives.md#secp-256-k1-addresses) mint output is an output that is owned by a collection of addresses.
+
+### **What SECP256K1 Mint Output Contains**
```

Comment:
Just a comment, not sure why ** are put on the section title. It only starts from this section.